### PR TITLE
Added maputils.py so the functions can be used in the future.

### PR DESCRIPTION
**Pull request by skaymen**
Contains maputils.py.

### CODE REVIEW

File: floodviz/maputils.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+def site_dict(site_list, url_prefix):
+    """Puts site data into a dictionary
+
+    Args:
+        site_list: A list of site ids to be queried on NWIS
+        url_prefix: The beginning of the NWIS url
+
+    Returns:
+        A dict containing various site information in a usable format.
```
Comment (mbucknell) [inline floodviz/maputils.py]: You are actually returning an array of dict.

File: floodviz/maputils.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import requests
+
+def site_dict(site_list, url_prefix):
+    """Puts site data into a dictionary
+
+    Args:
+        site_list: A list of site ids to be queried on NWIS
+        url_prefix: The beginning of the NWIS url
```
Comment (mbucknell) [inline floodviz/maputils.py]: You should also describe the type of url_prefix -- so something like "A string containing the NWIS site url".

File: floodviz/maputils.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        dnice.append(line_dict)
+
+    return dnice
+
+
+def write_geojson(filename, data):
+    """Writes site data to a .json file so it can be mapped
+
+       Args:
+           filename: the file to be written to
```
Comment (mbucknell) [inline floodviz/maputils.py]: Please describe the parameter types. Particularly important for data which is an array of dict.

File: floodviz/maputils.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+    with open(filename, "w") as f:
+        f.write("{ \"type\": \"FeatureCollection\", \"features\": [ \n")
+
+        for i, datum in enumerate(data):
+            f.write("{ \"type\": \"Feature\",\n \"geometry\": {\n \"type\": \"Point\",\n \"coordinates\" : "
+                    "[" + datum.get('dec_long_va') + ", " + datum.get('dec_lat_va') + "]\n },\n")
+            f.write(" \"properties\": {\n \"name\": \"" + datum.get('station_nm') + "\",\n \"id\": \""
+                    + datum.get('site_no') + "\",\n \"huc\": \"" +
+                    datum['huc_cd'] + "\" \n } \n }")
```
Comment (mbucknell) [inline floodviz/maputils.py]: Should use get for this one too.

File: floodviz/maputils.py (hunk 2 above)
Comment (skaymen) [inline floodviz/maputils.py]: Fixed.

File: floodviz/maputils.py (hunk 1 above)
Comment (skaymen) [inline floodviz/maputils.py]: Fixed.

File: floodviz/maputils.py (hunk 4 above)
Comment (skaymen) [inline floodviz/maputils.py]: Fixed.

File: floodviz/maputils.py (hunk 3 above)
Comment (skaymen) [inline floodviz/maputils.py]: Fixed.

**Pull request merged**